src/main/java/eu/yals/json/ErrorJson.java

```diff
@@ -55,28 +67,34 @@ public static ErrorJson createFromSetOfErrors(@NotNull Set<ConstraintViolation>
         return errorJson;
     }
 
-
-    public Error getError() {
-        return error;
-    }
-
-    public List<Error> getErrors() {
-        return errors;
-    }
-
-    @Since(1.0)
+    /**
+     * JSON field and error message combination.
+     */
     public static class Error {
-        @Since(1.0)
-        String field;
+        @Getter
+        private String field;
 
-        @Since(1.0)
-        String errorMessage;
+        @Getter
+        private String errorMessage;
 
+        /**
+         * Creates blank {@link Error} object.
+         *
+         * @return no params {@link Error}
+         */
         static Error create() {
             return new Error();
         }
 
-        static Error create(@NotNull ConstraintViolation violation) {
+        /**
+         * Create {@link Error} object from given {@link ConstraintViolation} object.
+         * Extracts it to field and message.
+         *
+         * @param violation not empty object, which contains field and error message
+         * @return error object with given params
+         */
+        @SuppressWarnings("rawtypes")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)